pallas-math/src/math.rs

```diff
@@ -49,6 +46,18 @@ pub trait FixedPrecision:
 
     /// Entry point for bounded iterations for comparing two exp values.
     fn exp_cmp(&self, max_n: u64, bound_self: i64, compare: &Self) -> ExpCmpOrdering;
+
+    /// Round to the nearest integer number
+    fn round(&self) -> Self;
+
+    /// Round down to the nearest integer number
+    fn floor(&self) -> Self;
+
+    /// Round up to the nearest integer number
+    fn ceil(&self) -> Self;
+
+    /// Truncate to the nearest integer number
+    fn trunc(&self) -> Self;
```

Comment:
Usually API like this also comes with a `must_use` warning. This helps the compiler warns the user if they are expecting to use the output of the function as the function does not modify the `self` in place.

For example: https://doc.rust-lang.org/1.81.0/src/core/num/mod.rs.html#546

```suggestion
    /// Round to the nearest integer number
    #[must_use]
    fn round(&self) -> Self;

    /// Round down to the nearest integer number
    #[must_use]
    fn floor(&self) -> Self;

    /// Round up to the nearest integer number
    #[must_use]
    fn ceil(&self) -> Self;

    /// Truncate to the nearest integer number
    #[must_use]
    fn trunc(&self) -> Self;
```